src/Sylius/Component/Taxonomy/Model/Factory/TaxonomyFactoryInterface.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Component\Taxonomy\Factory;
+
+
+/**
+ * @author Magdalena Banasiak <magdalena.banasiak@lakion.com>
+ */
+interface TaxonomyFactoryInterface
+{
+
```

Comment:
Why there is not method in this interface? Should be at least one (this implemented in ``TaxonomyFactory``)